src/idea.js

```diff
@@ -11,17 +11,16 @@ class Idea {
      localStorage.setItem("storedInformation", JSON.stringify(userIdeas));
 
    }
-
   deleteFromStorage() {
-    getStoredIdeas()
-    var elementId = parseInt(event.target.id, 10);
-    for (var i = 0; i < userIdeas.length; i++){
-      if (userIdeas[i].id === elementId){
-        userIdeas.splice(i, 1)
-        this.saveToStorage()
-      }  
+    // localStorage.clear();
```

Comment:
If there is a future need for these commented lines, an additional comment explaining why is best practice IMO, otherwise, I would consider removing until they are needed.

Reply:
Done and done! Thanks!